### PR TITLE
Updated the mission.html typos and styling

### PR DESCRIPTION
**Pull request by komalhadimani4444**
I have updated the typos in the about game section and also changed the styling of the text.

### CODE REVIEW

**Review by starlightknown — APPROVED** [review]
looks good to me, thank you :)

**Comment by starlightknown** [comment]
> looks good to me, thank you :)

noticed that you have added the text of twice or forgot to remove the previous one, please make changes

File: mission.html [hunk 1]
```diff
@@ -1,114 +1,129 @@
...
+      People may experience the following symptoms: <br>
+      <br>
+      Abnormal Body Posturing or Facial Expressions.<br>
+      Abnormal Tone of Voice.<br>
+      Avoidance of Eye Contact or Poor Eye Contact.<br>
+      Behavioral Disturbances.<br>
+      Deficits in Language Comprehension.<br>
+      Delay in Learning to Speak.<br>
+      Flat or Monotonous Speech.<br>
+      Inappropriate Social Interaction.<br>
```
Comment (starlightknown) [inline mission.html:116]: remove these extra kines, it's repeated. till line 157

Comment (starlightknown) [reply mission.html:116]: I have reverted the PR so could you please make a new one with the changes requested?

Comment (komalhadimani4444) [reply mission.html:116]: okay i will add

Comment (komalhadimani4444) [reply mission.html:116]: do i have to remove that using another pull request?

**Pull request merged**